### PR TITLE
updating example to use NuGet of DeepSpeechClient

### PR DESCRIPTION
**Pull request by willieNU**

### CODE REVIEW

File: net_framework/DeepSpeechWPF/App.xaml.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ protected override void OnStartup(StartupEventArgs e)
             {
                 //Register instance of DeepSpeech
                 DeepSpeechClient.DeepSpeech deepSpeechClient =
-                    new DeepSpeechClient.DeepSpeech("output_graph.pbmm");
```
Comment (carlfm01) [inline net_framework/DeepSpeechWPF/App.xaml.cs:23]: Maybe an extra comment here to put the link of the release page, like more model available...and put the link to releases.

**Comment by reuben** [comment]
We use the example on the master branch to test every PR. This one does not have to keep the reference.

**Comment by lissyx** [comment]
Is it okay to merge ?

**Comment by lissyx** [comment]
ping @willieNU @carlfm01 @reuben ?

**Comment by reuben** [comment]
Carlos left a comment that hasn't been addressed yet, but otherwise this should be OK to merge, the concern about DeepSpeech CI does not apply to this branch.

**Comment by lissyx** [comment]
Ok, let's merge this then.

**Pull request merged**